src/client/theme-default/Layout.vue

```diff
@@ -150,3 +154,34 @@ const pageClasses = computed(() => {
   ]
 })
 </script>
+
+<style>
+#ads-container {
+  margin: 0 auto;
+}
+
+@media (min-width: 420px) {
+  #ads-container {
+    position: relative;
+    right: 0;
+    float: right;
+    margin: -8px -8px 24px 24px;
+    width: 146px;
+  }
+}
+
+@media (max-width: 420px) {
+  #ads-container {
+    /* Avoid layout shift */
+    height: 105px;
```

Comment:
This is tough one, but should it be `min-height` instead? Because depending on the ads, it might exceed 105px...?

Reply:
but that would also create a layout shift, no? 🤔

Reply:
Well yeah... if the ad has long text, in that case it would cause layout shift for few pixels... not sure how often that occurs... but yeah, I think we do have enough space under the ad so it should be fine!